tldw_Server_API/app/api/v1/endpoints/admin.py

```
@@ -1170,52 +1083,29 @@ async def get_user_details(
-        else:
-            # SQLite
-            cursor = await db.execute(
-                "SELECT * FROM users WHERE id = ?",
-                (user_id,)
-            )
-            user = await cursor.fetchone()
-
+        repo = await AuthnzUsersRepo.from_pool()
+        user = await repo.get_user_by_id(user_id)
         if not user:
             raise UserNotFoundError(f"User {user_id}")
```

Ruff (0.14.6)

Abstract `raise` to an inner function

(TRY301)